src/queryMetrics/clientSideMetrics.ts

```diff
@@ -0,0 +1,58 @@
+import { QueryMetricsUtils } from "./queryMetricsUtils";
+
+export class ClientSideMetrics {
+    private requestCharge: number;
+    constructor(requestCharge: number) {
+        // Constructor
+        if (!QueryMetricsUtils.isNumeric(requestCharge)) {
```

Comment:
Is this check needed still?

Reply:
If we want to error on the client side, yes. TypeScript is a build time only type system, so it's possible to ignore it. Is this something we should aggressively guard against?

Reply:
Removed since this isn't meant to be used by customers, so we don't need to aggressively typecheck.